.github/workflows/openblas.yml

```diff
@@ -0,0 +1,144 @@
+# Minium test for Open3D with OpenBLAS/LAPACK on both x86_64 and aarch64.
+# This test is intentionally kept minimal and self-contained.
+
+name: Ubuntu OpenBLAS CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+env:
+  GCE_CLI_GHA_VERSION: '302.0.0'      # Fixed to avoid dependency on API changes
+
+jobs:
+  openblas_x86_64:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+    env:
+      BASE_IMAGE: ubuntu:20.04
+      CMAKE_VER: cmake-3.19.7-Linux-x86_64
+      CCACHE_TAR_NAME: open3d-x86_64-ci-ccache
+    steps:
+      - name: Cancel outdated
+        uses: fkirc/skip-duplicate-actions@master
+        with:
+          github_token: ${{ github.token }}
+
+      - name: Checkout source code
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Maximize build space
+        run: |
+          source util/ci_utils.sh
+          maximize_ubuntu_github_actions_build_space
+
+      - name: Docker build
+        run: |
+          docker build --build-arg BASE_IMAGE=${{ env.BASE_IMAGE }} \
+                       --build-arg CMAKE_VER=${{ env.CMAKE_VER }} \
+                       --build-arg CCACHE_TAR_NAME=${{ env.CCACHE_TAR_NAME }} \
+                       -t open3d-openblas-ci:latest \
+                       -f .github/workflows/Dockerfile.openblas .
+
+      - name: C++ tests
+        run: |
+          docker run --rm open3d-openblas-ci:latest build/bin/tests
+
+      - name: Python tests
+        run: |
+          docker run --rm open3d-openblas-ci:latest \
+            pytest python/test/ --ignore python/test/ml_ops/ --ignore python/test/t/io/test_realsense.py
+
+      # https://stackoverflow.com/a/34093828/1255535
+      - name: Copy ccache out from the container
+        run: |
+          docker run --rm --entrypoint \
+            cat open3d-openblas-ci:latest /${{ env.CCACHE_TAR_NAME }}.tar.gz \
+            > ${GITHUB_WORKSPACE}/${{ env.CCACHE_TAR_NAME }}.tar.gz
```

Comment:
Using a mounted volume should be faster than `cat`:
https://stackoverflow.com/a/49357803

Reply:
Done